react-node-mongodb/server/package.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Same here, update your dependencies and run an audit to be sure there isn't any security issues to fix (and there's some ^^)

Reply:
audit done